doc/man/scons.xml

```diff
@@ -3384,89 +3384,98 @@ object.</para>
 
 </refsect2>
 
-<refsect2 id='methods_and_functions_to_do_things'>
-<title>Methods and Functions To Do Things</title>
+<refsect2 id='env_methods'>
+<title>&SCons; Functions and Environment Methods</title>
 
-<para>In addition to Builder methods,
-&scons;
-provides a number of other &consenv; methods
-and global functions to
-manipulate the build configuration.
-Usually, a &consenv; method
-and global function with the same name both exist
-for convenience.
-In the following list, the global function
-is documented in this style:</para>
+<para>
+&SCons; provides a variety of  &consenv; methods
+and global functions to manipulate the build configuration.
+Often, a &consenv; method and a global function with
+the same name exist for convenience.
+In this section, both forms are shown if the function can be called
+in either way.
+The documentation style for these is as follows:
+</para>
 
 <programlisting language="python">
-<function>Function</function>(<parameter>arguments, [optional arguments]</parameter>)
+<function>Function</function>(<parameter>arguments, [optional arguments, ...]</parameter>)  # Global function
+<replaceable>env</replaceable>.<methodname>Function</methodname>(<parameter>arguments, [optional arguments, ...]</parameter>)  # Environment method
 </programlisting>
 
-<para>and the &consenv; method looks like:</para>
-
-<programlisting language="python">
-<replaceable>env</replaceable>.<methodname>Function</methodname>(<parameter>arguments, [optional arguments]</parameter>)
-</programlisting>
+<para>
+In these function signatures,
+arguments in brackets (<literal>[]</literal>) are optional,
+and ellipses (<literal>...</literal>) indicate possible repetition.
+Positional vs. keyword arguments are usually detailed
+in the following text, not in the signature itself.
+The &Python; positional-only (<literal>/</literal>)
```

Comment:
I'd remove this bit `The Python positional-only (/) and keyword-only (*)
markers are not used.`  We never refer to using that notation anywhere, so I think no need to say it's not needed here, and for the python neophyte it might just cause confusion.

Reply:
I can demote it to a comment - want to keep a placeholder since it's possible the "not used" could change in future - there are some functions we can't correctly describe in the synopsis without using more Pythonic syntax, and have to leave to prose.  

It came up for me because when I was working on that, the Python doc folks had just decided to reverse a previous policy of not using those special markers because "it might be confusing", in favor of being able to provide a more correct synopsis.  Python docs, of course, are intended for Python users and we still suggest you don't need to know (much) Python so it's not quite the same. Though I still have a preference for having the manpage describe things "completely", which includes the synopsis of an official interface.